src/app/todoList/todoList.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core';
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Component({
+  selector: 'app-todoList',
+  templateUrl: './todoList.component.html',
+  styleUrls: ['./todoList.component.css']
+})
+
+export class TodoListComponent {
+  newTodoTitle: string = '';
+  todos: Array<Todo> = [];
+
+  remain() {
+    let il = 0;
+    this.todos.forEach(t => {
```

Comment:
`forEach` nie jest tutaj raczej poprawne semantycznie. Ono jest bardziej po to, żeby wykonać coś na każdym elemencie. Tutaj lepiej byłoby `for ... of`